Makefile

```diff
@@ -1,12 +1,15 @@
-SHELL=/bin/bash
+SHELL=/bin/sh
```

Comment:
I'm not sure we should cling to POSIX sh instead of adopting bash.

Reply:
I agree, but I wanted to defer this decision; hence #396.